nextflow.config

```diff
@@ -14,7 +14,7 @@ params {
     mode = "fastq"
     miairr="$projectDir/assets/reveal/mapping_MiAIRR_BioSample_v1.3.1.tsv"
     index_file = false
-
+    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/'
```

Comment:
```suggestion
    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/airflow/'
```

Reply:
Just as a warning, if this change is applied then `airflow` would need to be removed from all the test configs files

Reply:
Thanks, I've updated both accordingly!